packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -2,6 +2,10 @@
 // Licensed under the MIT license.
 
 import { memoizeFunction, Stack, useTheme } from '@fluentui/react';
+/* @conditional-compile-remove(PSTN-calls) */
+import { useState } from 'react';
+// uncomment this to test dialpad dtmf
+// import { DefaultButton } from '@fluentui/react';
```

Comment:
This should not be in this PR

Reply:
This is for the dummy button to activate dialpad. need to uncomment those lines in order to test it. Comment them out so we are not introducing UI changes

Reply:
Updated PR to remove all code for the dummy button